.gitignore

```diff
@@ -21,4 +21,6 @@ target
 *_pb2.py
 build
 dist
-*.egg-info
\ No newline at end of file
+*.egg-info
+*.ipynb_checkpoints
+docs/examples/*.csv
```

Comment:
`.gitignore` is used to ignore files that are widely considered either hidden or temporary. It seems safer not to include program-specific files like this.